GWAS/PCA.ipynb

```diff
@@ -1122,9 +1127,15 @@
     "    library(ggplot2)\n",
     "    library(gridExtra)\n",
     "    library(matrixStats)\n",
+    "    library(stringr)\n",
```

Comment:
same comment as before -- please remove this and also remove the `print()` statement. This pipeline is meant to be a generic software.

Reply:
and if you dont need stringr after removing it then please also remove stringr from loading.